packages/block-editor/src/components/rich-text/editable.js

```diff
@@ -119,6 +119,18 @@ export default class Editable extends Component {
 			this.editorNode.className = classnames( nextProps.className, CLASS_NAME );
 		}
 
+		if ( this.props.start !== nextProps.start ) {
+			this.editorNode.setAttribute( 'start', nextProps.start );
```

Comment:
Why this node gets updated using setAttribute but in other places the value is set through direct mutation?

Reply:
Probably because the others need to be removed if needed. Would be good to unify this though.